src/MultiLogging.hs

```diff
@@ -0,0 +1,173 @@
+{-# LANGUAGE ApplicativeDo #-}
```

Comment:
What do you think of adding something like: "This module exposes functions to help log to multiple FastLogger loggers, using an API similar to monad-log (which is itself also used)."